tests/api_tests/users/test_views.py

```diff
@@ -380,3 +372,17 @@ def test_get_404_path_nodes_user_id_no_user(self):
         url = "/{}nodes/{}/".format(API_BASE, self.user_one._id)
         res = self.app.get(url, expect_errors=True)
         assert_equal(res.status_code, 404)
+
+
+class TestDeactivatedUser(ApiTestCase):
+
+    def setUp(self):
+        super(TestDeactivatedUser, self).setUp()
+        self.user = AuthUserFactory()
+        self.user.is_disabled = True
+        self.user.save()
+
+    def test_return_deactivated_user(self):
+        self.url = '/{}users/{}/'.format(API_BASE, self.user._id)
+        res = self.app.get(self.url, auth=self.user.auth , expect_errors=True)
```

Comment:
It might be good to have a "sanity check" assertion that tests that an error response is not returned if the same user is not disabled.

I can add this when I merge.